Use "+=" operator for LDFLAGS to be able to add others option to it.

diff --git a/linux/Makefile b/linux/Makefile
index 5a49d81..38a336c 100644
--- a/linux/Makefile
+++ b/linux/Makefile
@@ -19,7 +19,6 @@ include $(MAKEDIR)/syslinux.mk
 OPTFLAGS = -g -Os
 INCLUDES = -I$(SRC) -I$(objdir) -I$(SRC)/../libinstaller
 CFLAGS	 = $(GCCWARN) -D_FILE_OFFSET_BITS=64 $(OPTFLAGS) $(INCLUDES)
-LDFLAGS	 = 
 
 SRCS     = syslinux.c \
 	   ../libinstaller/syslxopt.c \
diff --git a/mk/build.mk b/mk/build.mk
index 0ca82be..b71dc10 100644
--- a/mk/build.mk
+++ b/mk/build.mk
@@ -20,7 +20,6 @@ OPTFLAGS   = -g -Os
 INCLUDES   =
 CFLAGS     = -W -Wall -Wno-sign-compare -D_FILE_OFFSET_BITS=64 \
              $(OPTFLAGS) $(INCLUDES)
-LDFLAGS    =
 LIBS	   =
 
 .SUFFIXES: .c .o .S .s .i .elf .com .bin .asm .lst .c32 .lss
diff --git a/mk/com32.mk b/mk/com32.mk
index aeb8a2c..dcc2bd5 100644
--- a/mk/com32.mk
+++ b/mk/com32.mk
@@ -80,7 +80,7 @@ SFLAGS     = $(GCCOPT) $(GCCWARN) \
 	     -I$(topdir)/core/include
 
 COM32LD	   = $(com32)/lib/$(ARCH)/elf.ld
-LDFLAGS    = -m elf_$(ARCH) -shared --hash-style=gnu -T $(COM32LD)
+LDFLAGS    += -m elf_$(ARCH) -shared --hash-style=gnu -T $(COM32LD)
 LIBGCC    := $(shell $(CC) $(GCCOPT) --print-libgcc)
 
 LNXCFLAGS  = -I$(com32)/libutil/include $(GCCWARN) -O -g \
diff --git a/mk/efi.mk b/mk/efi.mk
index dc2b708..bcdeec8 100644
--- a/mk/efi.mk
+++ b/mk/efi.mk
@@ -43,7 +43,7 @@ CFLAGS = -I$(EFIINC) -I$(EFIINC)/$(EFI_SUBARCH) \
 CRT0 := $(LIBDIR)/crt0-efi-$(EFI_SUBARCH).o
 LDSCRIPT := $(LIBDIR)/elf_$(EFI_SUBARCH)_efi.lds
 
-LDFLAGS = -T $(SRC)/$(ARCH)/syslinux.ld -Bsymbolic -pie -nostdlib -znocombreloc \
+LDFLAGS += -T $(SRC)/$(ARCH)/syslinux.ld -Bsymbolic -pie -nostdlib -znocombreloc \
 		-L$(LIBDIR) --hash-style=gnu -m elf_$(ARCH) $(CRT0) -E
 
 SFLAGS     = $(GCCOPT) $(GCCWARN) $(ARCHOPT) \
diff --git a/mk/elf.mk b/mk/elf.mk
index b46dbd0..46d7979 100644
--- a/mk/elf.mk
+++ b/mk/elf.mk
@@ -67,7 +67,7 @@ GCCOPT += -mregparm=3 -DREGPARM=3
 endif
 
 SFLAGS     = $(GCCOPT) -D__COM32__ -D__FIRMWARE_$(FIRMWARE)__ 
-LDFLAGS    = -m elf_$(ARCH) -shared --hash-style=gnu -T $(com32)/lib/$(ARCH)/elf.ld --as-needed
+LDFLAGS    += -m elf_$(ARCH) -shared --hash-style=gnu -T $(com32)/lib/$(ARCH)/elf.ld --as-needed
 LIBGCC    := $(shell $(CC) $(GCCOPT) --print-libgcc)
 
 LNXCFLAGS  = -I$(com32)/libutil/include -W -Wall -O -g -D_GNU_SOURCE
diff --git a/mk/lib.mk b/mk/lib.mk
index f3fb07c..7af96f1 100644
--- a/mk/lib.mk
+++ b/mk/lib.mk
@@ -203,7 +203,7 @@ CORELIBOBJS = \
 	$(LIBENTRY_OBJS) \
 	$(LIBMODULE_OBJS)
 
-LDFLAGS	= -m elf_$(ARCH) --hash-style=gnu -T $(com32)/lib/$(ARCH)/elf.ld
+LDFLAGS	+= -m elf_$(ARCH) --hash-style=gnu -T $(com32)/lib/$(ARCH)/elf.ld
 
 .SUFFIXES: .c .o .a .so .lo .i .S .s .ls .ss .lss
 
diff --git a/utils/Makefile b/utils/Makefile
index b103124..f30c46e 100644
--- a/utils/Makefile
+++ b/utils/Makefile
@@ -20,7 +20,7 @@ include $(MAKEDIR)/syslinux.mk
 CC_FOR_BUILD ?= $(CC)
 
 CFLAGS   = $(CFLAGS_FOR_BUILD) $(GCCWARN) -Os -fomit-frame-pointer -D_FILE_OFFSET_BITS=64 -I$(SRC)
-LDFLAGS  = $(LDFLAGS_FOR_BUILD) -O2
+LDFLAGS  += $(LDFLAGS_FOR_BUILD) -O2
 
 C_TARGETS	 = isohybrid gethostip memdiskfind
 SCRIPT_TARGETS	 = mkdiskimage
